### PR TITLE
A collection of minor updates for 2022

### LINKED ISSUE #177: Fix installations errors
- [x] Need `branca` for `07.01-ipycanvas`
- [x] Need `pyvista` for `07.04-ipygany`

### PR DESCRIPTION
**Pull request by mwcraig**
This also contains a couple of fixes for ipywidgets 8. Closes #177

### CODE REVIEW

**Comment by marimeireles** [comment]
On it! :)

**Comment by marimeireles** [comment]
Hey @mwcraig is not working at all to me.
I had a problem with `couldn't find version 0.2.19 of voila` so I changed to the latest `voila==0.3.5` and had a lot of conflicts. Changed back to `0.2.10` and looks fine. We surely need the 2.19?

**Comment by ianhi** [comment]
The readme claims that: 

> The installation instructions were tested on an up-to-date version of Windows 10 Professional. If you encounter any issues on Windows please open an issue or contact us through slack.

Is that actually true? Does anyone have windows on which to test this?

File: environment.yml [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+  - python=3.9
   - pythreejs=2.3.0
-  - pyvista=0.31.3
+  - pyvista=0.34.0
   - requests
   - scikit-image
   - scipy
   - sidecar=0.5.1
-  - voila=0.2.10
+  - voila=0.2.19
```
Comment (ianhi) [inline environment.yml:29]: The last in the `0.2.x` series is 16

```suggestion
  - voila=0.2.16
```

File: environment.yml [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
+  - branca=0.5.0
+  - ipycanvas=0.12.0
+  - ipycytoscape=1.3.3
+  - ipydatagrid=1.1.12
+  - ipyevents=2.0.1
   - ipygany=0.5.0
-  - ipyleaflet=0.14.0
-  - ipympl=0.7.0
+  - ipyleaflet=0.16.0
+  - ipympl=0.9.1
```
Comment (ianhi) [inline environment.yml:13]: I will need to make a new release of ipympl if we switch to ipywidgets 8. The current release has a gate on `ipywidgets<8`.
I started work on that here: https://github.com/matplotlib/ipympl/pull/461 but stalled out bc I couldn't get the tests all passing

Comment (mwcraig) [reply environment.yml:13]: All of these will need new releases for ipywidgets 8 -- the pins should make sure that no one gets an 8-only package.

One reason for setting the goal of presenting in 8 is to help work through the remaining issue.

**Comment by mwcraig** [comment]
> Hey @mwcraig is not working at all for me. I had a problem with `couldn't find version 0.2.19 of voila` so I changed to the latest `voila==0.3.5` and had a lot of conflicts. Changed back to `0.2.10` and looks fine. We surely need the 2.19?

Thanks -- it turns out we can use 0.3.5 if we drop `voila-vuetify`, which I think is fine, so I made that change.

**Comment by mwcraig** [comment]
> The readme claims that:
> 
> > The installation instructions were tested on an up-to-date version of Windows 10 Professional. If you encounter any issues on Windows please open an issue or contact us through slack.
> 
> Is that actually true? Does anyone have windows on which to test this?

It will be in about 30 minutes -- I have a windows 10 VM on my work laptop.

File: environment.yml (hunk 1 above)
Comment (mwcraig) [reply environment.yml]: Thanks -- ended up changing it to `0.3.5` and dropping `voila-vuetify` and `voila-material`

**Pull request merged**